.eslintrc

```diff
@@ -14,7 +14,8 @@
   },
   "extends": ["fbjs/opensource"],
   "rules": {
-    "linebreak-style": 0
+    "linebreak-style": 0,
+    "indent": ["error", 2]
```

Comment:
In the future, it'd be best not to mess with linter settings in the same PR as other work, unless it's directly needed to make new source format nicely (for example). I'm fine with this one though :smile: